docs/learn.html

```diff
@@ -50,6 +50,7 @@ <h5 class="my-2">
               <a href="vsc-getting-started.html">VS Code</a>,
               <a href="eclipse-getting-started.html">Eclipse</a>, or
               <a href="intellij-getting-started.html">IntelliJ</a>.
+              <a href="gettingstarted-overview.html">Read more</a> about local, remote and browser-based configurations.
```

Comment:
Add comma after remote in this new sentence so it reads:
"Read more</a> about local, remote, and browser-based configurations. "